qiskit/circuit/library/arithmetic/polynomial_pauli_rotations.py

```diff
@@ -153,7 +153,7 @@ class PolynomialPauliRotations(FunctionalPauliRotations):
 
     .. math::
 
-        p(x) = \sum_{j=0}^{j=d} c_j x_j
+        p(x) = \sum_{j=0}^{j=d} c_j x^j
```

Comment:
Thank you for you review. It makes sense what you are saying since you can write LinearPauliRotations using PolynomialPauliRotations with a 1st degree Polynomial.
I'll have a look into that and update accordingly.

Reply:
Given the code below (inside `test_functional_pauli_rotations.py` at link: https://github.com/Qiskit/qiskit-terra/blob/3763e61f16f97598cb0fa317277ad85fa15db738/test/python/circuit/library/test_functional_pauli_rotations.py#enroll-beta):
```py
    def test_polynomial_function(self, coeffs, num_state_qubits):
        """Test the polynomial rotation."""

        def poly(x):
            res = sum(coeff * x**i for i, coeff in enumerate(coeffs))
            return res

        polynome = PolynomialPauliRotations(num_state_qubits, [2 * coeff for coeff in coeffs])
        self.assertFunctionIsCorrect(polynome, poly)
```
As @Cryoris was suggesting, maybe the correct way to approach this problem is to only rename `cos(p(i)/2)` in order to being allined with the code above.

Otherwise we can use `c_j /2 x^j` notation but change the following text inside the current #9922 pull request at line 158:
```
where the input coefficients: ``coeffs``= :math:`c` \2.
```